services/web/project/blueprint/github_repository_blueprint.py

```diff
@@ -0,0 +1,24 @@
+from flask import Blueprint
+from ..service import github_repository_extrator_service as service
+from ..repository import github_repository_repository as repository
+from ..serializer import GithubRepositorySchema
+
+
+bp_github_repository = Blueprint('github_repository', __name__)
+
+@bp_github_repository.route('/github-repository', methods=['GET'])
+def list_all():
+    schema = GithubRepositorySchema(many = True)
+    result = repository.list_all()
+    return schema.jsonify(result), 200
+
+@bp_github_repository.route('/github-repository/<id>', methods=['GET'])
+def get(id):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'id'](https://app.codacy.com/manual/mpgabriel95/TCC/pullRequest?prid=5239659)